src/pages/ProductDetail/components/CommentItem/CommentItem.tsx

```diff
@@ -18,7 +18,21 @@ const elapsedTime = (date) => {
 	return `${Math.floor(days)}일 전`;
 };
 
-function CommentItem({item}) {
+interface Comment {
+    id:number;
+    writer:Writer;
+    content:string;
+    createdAt:string;
+    updatedAt:string;
+  }
+
+interface Writer {
+    image: string;
+    nickname:string;
+    id:number;
+}
```

Comment:
P3:
writer 타입을 따로 정의해주신 점이 좋습니다~
다만 대소문자로만 구분이 되니 타입 이름을 지으실때 좀 더 타입임을 알 수 있게 해주시면 좋을 것 같아요.
Writer 라는 이름은 writer 관련 정보를 보여주는 컴포넌트일 수도 있으니까요~

```suggestion
interface WriterProps {
    image: string;
    nickname:string;
    id:number;
}
```

Reply:
다른 타입들도 동일합니다~ isActive처럼 이름이 어떤 값인지 알 수 있게 변수도, 타입도 네이밍하시는 것을 추천드려요

ex. isActive, CommentProps, DirectionType